interpreter/cling/include/cling/Interpreter/Interpreter.h

```diff
@@ -771,6 +772,12 @@ namespace cling {
     const InterpreterCallbacks* getCallbacks() const {return m_Callbacks.get();}
     InterpreterCallbacks* getCallbacks() { return m_Callbacks.get(); }
 
+    ///\brief Returns the JIT managed by the Interpreter.
+    /// Accesses and returns the JIT held in the IncrementalJIT instance
+    /// managed by m_Executor
+    const llvm::orc::LLJIT* getExecutionEngine() const;
+    llvm::orc::LLJIT* getExecutionEngine();
```

Comment:
LGTM. But why do you need both const and non-const versions of `getExecutionEngine()`?
I'll wait for @vgvassilev or @hahnjo to comment before merging.